src/data/full-quiz.js

```diff
@@ -1,4 +1,22 @@
 const fullQuiz = [
+  {
+    Question: "What is the function of resize2fs command?",

```

Comment:
I think we should add the word `the` before the resize2fs

Proposed fix:
What is the function of the resize2fs command?